ci-operator/step-registry/openshift/e2e/openstack/nfv/openshift-e2e-openstack-nfv-workflow.yaml

```
@@ -0,0 +1,18 @@
+workflow:
+  as: openshift-e2e-openstack-nfv
+  steps:
+    pre:
+      - chain: ipi-openstack-pre
+    test:
+      - ref: openshift-e2e-test
+    post:
+      - chain: ipi-openstack-post
+    env:
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      CONFIG_TYPE: "minimal"
+      TEST_SKIPS: The HAProxy router should be able to connect to a service that is
```

```suggestion
      TEST_SKIPS: |
         The HAProxy router should be able to connect to a service that is
```
nipitck: so that once this skip is removed the diff would look cleaner